ci-operator/step-registry/gcp/deprovision/buildhost/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+- gpei
+- jianlinliu
+- jianli-wei
+- rphillips
```

Comment:
I guess you can drop the `gpei`, `jianlinliu` and `jianli-wei` from the OWNER list for the `buildhost` ref.